index.html

```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
     </main>
   </body>
 </html>
+
+
+      <!-- My works begins here -->
+
+      <section class="my-works">
+        <h2 class="section-title">
+          My Recent Works
+          <hr class="horizontal-line" />
+        </h2>
+
+        <div class="works-outer-wrapper">
+          <ul class="works-inner-wrapper">
+            <li>
+              <div class="first-card">
+                <img
+                  class="first-work"
+                  src="images/first_work.svg"
+                  alt="First Work"
+                />
+                <div class="card-content">
+                  <h3 class="card-title">Multi-Post Stories</h3>
+                  <p class="card-description">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                  </p>
+                  <ul class="card-list">
+                    <li class="card-list-item">css</li>
+                    <li class="card-list-item">html</li>
+                    <li class="card-list-item">bootstrap</li>
+                    <li class="card-list-item">Ruby</li>
+                  </ul>
+                  <a class="card-link" href="#">See Project</a>
```

Comment:
- [ ] The buttons should follow the button styling: Active, Hover, and Pressed. Please kindly add it for all the buttons 🙃 😉 

<img width="586" alt="Screen Shot 2022-10-25 at 21 02 24" src="https://user-images.githubusercontent.com/74501523/197859697-23aeb23f-4283-4600-a884-fdbf680832cf.png">

Reply:
@cynthiainga I don't understand this comment. Taking a look at the buttons in the Figma design, the buttons are either bordered with a color (1px solid #8993A4) which I used, a button with a background color (#FF6B00) which I used or a button with (background: rgba(255, 255, 255, 0.24)) which I used also.

Reply:
Hi @kennedyowusu, the styling: Active, Hover, and Pressed means that when for example users press the buttons change the color, and have the color of the pressed button, or when the button is disabled it takes the color of the disabled button and so one... I think it's clear now 🙃